aws/logs_monitoring/lambda_function.py

```diff
@@ -444,6 +464,16 @@ def batch(self, logs):
         return batches
 
 
+def compress_logs(batch, level):
+    if level < 0:
+        compression_level = 0
+    elif level > 9:
+        compression_level = 9
+    else:
+        compression_level = level
+
+    return gzip.compress(bytes(batch, 'utf-8'), compression_level)
```

Comment:
Can this method throw an error ? If so, we probably want to catch it here https://github.com/DataDog/datadog-serverless-functions/pull/190/files#diff-618731ddf1c446ef45d09613cb4b189bR339 to avoid retrying indefinitely